x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/footer.tsx

```diff
@@ -92,12 +89,7 @@ export const EventDetailsFooter = React.memo(
       onAddEventFilterClick,
     } = useEventFilterModal();
 
-    const { alertsEcsData } = useFetchEcsAlertsData({
-      alertIds: eventIds,
-      skip: expandedEvent?.eventId == null,
-    });
-
-    const ecsData = get(0, alertsEcsData);
+    const ecsData = expandedEvent.ecsData;
```

Comment:
we already have ecs from the onClick that expands the event

Reply:
yeah, I found that we can pass ecsData when expanding event in alerts table, but in Case details view I just couldn't get ecsData the same way. Therefor I removed the ecsData from expandedEvent and fetch it anyway in `x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/footer.tsx` Line 95.

Reply:
update: I ran it and still found that removing `useFetchEcsAlertsData` wouldn't work for case details view, so I think we still need to fetch it here